src/include/block_chunkcache.h

```diff
@@ -0,0 +1,74 @@
+/*-
+ * Copyright (c) 2014-2020 MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *  All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * WiredTiger's chunk cache. Locally caches chunks of remote objects.
+ */
+
+#define WT_CHUNKCACHE_DEFAULT_HASHSIZE 32 * 1024
+#define WT_CHUNKCACHE_DEFAULT_CHUNKSIZE 1024 * 1024
+#define WT_CHUNKCACHE_FILE 1
+#define WT_CHUNKCACHE_IN_VOLATILE_MEMORY 0
+#define WT_CHUNKCACHE_MINHASHSIZE 64
+#define WT_CHUNKCACHE_MAXHASHSIZE 1024 * 1024
+#define WT_CHUNKCACHE_MAX_RETRIES 1024 * 1024 * 1024
+#define WT_CHUNKCACHE_NAMEMAX 50
+#define WT_CHUNKCACHE_UNCONFIGURED 0
+
+struct __wt_chunkcache_hashid {
+    char objectname[WT_CHUNKCACHE_NAMEMAX];
+    uint32_t objectid;
```

Comment:
Since we're currently using `object` for tiered storage this is a bit confusing. What about `chunkname` and `chunkid` instead? And some comments about what these fields mean would be helpful.